README.md

```diff
@@ -9,6 +9,10 @@ Date.first_day_of_month_for(dat,day)
 - dat can be, date object / date string
 - day can be, case insensitive string of weekdays / below mentioned numbers representing weekdays
 
+Date.wednesday_of_year(week, year)
```

Comment:
Hey, sorry for late response. However, i appreciate your effort to make it better.
But, 
1. I wonder why only next_wednesday or wednesday_of_year?
2. Any chance that we add same methods dynamically for all the days of week? 
Please give it a thoguht and let me know your ideas.